### ISSUE TITLE
Conflict with ExpressiveCode

### ISSUE THREAD

**Issue by maxcelos** [HiDeoo/starlight-image-zoom#9]
### Describe the bug

After enabling this plugin in Starlight config, the `frame="none"` flag in my code blocks stoped working and the window frames are displayed.

### To Reproduce

1. Have a page with a code block that applies the `frame="none"` flag.
Example: 
![CleanShot 2024-05-18 at 13 08 20](https://github.com/HiDeoo/starlight-image-zoom/assets/15364289/c430bc83-0f9d-446e-9b63-7df1645826e8)

![CleanShot 2024-05-18 at 13 08 45](https://github.com/HiDeoo/starlight-image-zoom/assets/15364289/8477283c-6eb4-4dfd-9394-930947d7b0b7)

2. Install and enable the starlightImageZoomPlugin in `astro.config.mjs`
![CleanShot 2024-05-18 at 13 09 24](https://github.com/HiDeoo/starlight-image-zoom/assets/15364289/4b38ddfb-d36c-4785-bcb2-ef100d1d0b76)

3. The code block have window frames now
![CleanShot 2024-05-18 at 13 09 47](https://github.com/HiDeoo/starlight-image-zoom/assets/15364289/2c91754e-3ec0-41fe-98a9-a86606600ed9)

### Expected behavior

Should not make difference in code blocks

### How often does this bug happen?

Every time

### System Info

OS: Mac
Browser: Chrome
Plugin version: 0.4.0

**Comment by HiDeoo** [comment]
Thanks for your report 🙌 

Would you be able to share a small repro, either using [astro.new](https://astro.new/latest) or in a GitHub repository?

I tried setting a repro by adding `frame="none"` to the first code block in `docs/src/content/docs/demo.mdx` in this repository and running `pnpm dev` in the `docs/` directory while navigating to `http://localhost:4321/demo/` and everything seems to work as expected. The code block is rendered without a frame and the plugin is working as expected.
Also tried in a setup with the latest Starlight and Astro version, this seems to work as expected too.

**Comment by maxcelos** [comment]
Here it goes:

```
git clone https://github.com/maxcelos/bug-demo.git

cd bug-demo

pnpm install

pnpm dev
```

Go to the page `/guides/example/`. At the bottom of the page there is a sh code block. You will see it with window frame.

To test the issue, you can comment and uncomment the plugins config in astro.config.mjs

**Comment by HiDeoo** [comment]
Thanks for the repro, super appreciated 🙌 

I won't be able to investigate tonight as it's getting late over here, but until then, it looks like it's only affecting pure Markdown file so switching the extension from `.md` to `.mdx` seems to workaround the issue.

I'll try to take a look at this tomorrow morning.

**Comment by maxcelos** [comment]
You're right! Just changed the extension and the issue is gone. 

Maybe just place a heads up in documentation. Not sure if anyone will feel hurt by being obligated to use mdx. Personally it's not a problem :)

**Comment by HiDeoo** [comment]
I just published [`v0.5.0`](https://github.com/HiDeoo/starlight-image-zoom/releases/tag/v0.5.0) of the plugin which should now fix the issue :tada:

Thanks again for the detailed report and reproduction, it was very helpful!
